ci-operator/config/maistra/istio-operator/maistra-istio-operator-maistra-3.0.yaml

```diff
@@ -158,13 +158,13 @@ tests:
     - as: integration-ocp
       cli: latest
       commands: |
-        export QUAY_PASS=$(cat /maistra-secrets/password)
+        oc patch configs.imageregistry.operator.openshift.io/cluster --patch '{"spec":{"defaultRoute":true}}' --type=merge && sleep 5 
+        export DOCKER_INSECURE_REGISTRIES=$(oc get route default-route -n openshift-image-registry --template='{{ .spec.host }}')
```

Comment:
Is this route somewhat guessable? How is it formed? Is  it `<cluster_url>/<namespace>/some_name`? If we can guess the url we don't need to patch+wait here. We just need to export this variable. The patch+wait can be done inside the test (as it's already done).

Plus, we can remove the `credentials` section below (see my previous comment)

You still working on `make jobs`?

Reply:
Yes, it's guessable. I'll make some changes. I spent some time this morning setting a fedora VM for personal use so I'll be able and I set make an update successfully